var/spack/repos/builtin/packages/mgis/package.py

```diff
@@ -113,8 +125,13 @@ def cmake_args(self):
             args.append("-DPYTHON_LIBRARY={0}".format(python.libs[0]))
             args.append("-DPYTHON_INCLUDE_DIR={0}".format(python.headers.directories[0]))
             args.append("-DPython_ADDITIONAL_VERSIONS={0}".format(python.version.up_to(2)))
-            # adding path to boost
-            args.append("-DBOOST_ROOT={0}".format(self.spec["boost"].prefix))
+
+            if "py-pybind11" in self.spec:
```

Comment:
...and distinguish based on `@3.1:` or not

Reply:
ok